index.html

```diff
@@ -1,246 +1,139 @@
+            </ul>
         </div>
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
+        <a href="#" class="promore">SEE THE WHOLE PROGRAM</a>
+        <button type="button" class="probtn">Join Microverse</button>
+    </section>
 
-          <div class="feature-card">
-            <h3>Lecturer3</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-          <div class="feature-card">
-            <h3>Lecturer</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
+    <!--Added with Javscript Dynamically-->
+    <section class="featuredspeakers">
+      <h2 class="speakerhead">Featured Speakers</h2>
+    </section>
 
-        <div class="dis-none">
-          <button class="show-less" type="button">LESS &#x22C0;</button>
+    <footer class="footer">
+        <div class="partners">
+            <h2 class="partnerhead">Partner</h2>
+            <div class="partnerlogo">
+                <img class="plogo" src="img/tech.png" alt="Tech Crunch logo">
+                <img class="plogo" src="img/EdSurgelogo.png" alt="EdSurge logo">
+                <img class="plogo" src="img/CincoDíaslogo.png" alt="Cinco Días logo">
+                <img class="plogo" src="img/Infobaelogo.png" alt="Infobae logo">
+            </div>
         </div>
-      </div>
-    </section>
-    <div class="footer-class">
-      <div class="prg1">
-        <h3 class="program">Partners</h3>
-        <hr class="underline" />
-        <a class="bold">mozilla</a>
-        <a class="faint">Google</a>
-        <a class="bold">NAVER</a>
-        <a class="faint">daumkakao</a>
-        <a class="bold"><i class="fa fa-airbnb"></i>airbnb</a>
-      </div>
-    </div>
-    <footer class="footer-class1">
-      <div class="footer-content">
-        <img src="./images/creative-commons-license-logo.jpeg" alt="logo" />
-        <div class="footer-text">
-          <p>Copyright © 2015 Creative Commons Korea. Some rights reserved.</p>
-          <p class="licence-text">
-            This work is licensed under a
-            <a
-              rel="license"
-              href="http://creativecommons.org/licenses/by-nc/4.0/"
-              >Creative Commons Attribution-NonCommercial 4.0 International
-              License</a
-            >
-          </p>
+        <div class="bfooter">
+                <img class="blogo" src="img/techcrunchlogo.png" alt="Tech Crunch logo">
+            <div class="bfootertext">
+                <h4>2022 Microverse. Some rights reserved.</h4>
+                <p>Tech Crunch in association with Microverse.</p>
+            </div>
```

Comment:
- [ ] Great job so far, however, it would be great if you centre the footer, **both in the index and the about page**, horizontally so that it's in keeping with the design.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/169029402-34693f3a-edb6-488a-8d2d-c01cd08cf091.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169029490-bf94b642-3aa8-4e84-8ba0-2ebf7136c82c.png)